onnxruntime/test/testdata/transform/model_parallel/bart_mlp_megatron_basic_test.py

```diff
@@ -19,7 +19,7 @@
 dropout_np_vals = np.asarray([0.1], dtype=np.float32).reshape(())
 dropout_initializer = numpy_helper.from_array(dropout_np_vals, "ratio")
  
-dropout_mode_np_vals = np.array([True], dtype=np.bool).reshape(())
+dropout_mode_np_vals = np.array([False], dtype=np.bool).reshape(())
```

Comment:
nit: I thought our test is using InferenceSession to run the graph, right? If so, I assume dropout will by default be ingnored in inference path, am I correct?

Reply:
It used to be so, but when i rebased with current master, the partition correctness tests started failing with different results before and after combining the graphs. This got fixed when i set the training mode to false. Hence changed, since we do expect the dropout to be ignored in this test.